CONTRIBUTING.md

```diff
@@ -24,6 +24,7 @@ The following environment variables must be set:
 - XOA_TEMPLATE - A VM template that has an existing OS **already installed**
 - XOA_DISKLESS_TEMPLATE - A VM template that does not have an existing OS (found from `xe template-list`)
 - XOA_ISO - The name of an ISO that exists on the same pool as `XOA_POOL`
+- XOA_ISO_SR - The name of an ISO storage repository that exists on the same pool as `XOA_POOL`
```

Comment:
It also needs to be writable from the hosts.
So a read only NFS ISO sr won't work.